.travis.yml

```diff
@@ -34,6 +34,12 @@ before_script:
 
 after_success:
   - codecov
+  - retriever  install postgres FrayJorge -u postgres -d testdb
```

Comment:
I added some lines to test direct retriever installation. 
We can remove them if not needed